sample/build.gradle

```diff
@@ -8,6 +8,7 @@ repositories {
 }
 
 android {
+    namespace "com.yalantis.ucrop.sample"
```

Comment:
Minor (🔍): Now that this is defined here, this [package="com.yalantis.ucrop.sample"](https://github.com/Automattic/uCrop/blob/develop-non-native/sample/src/main/AndroidManifest.xml#L3) definition could be deleted from the [AndroidManifest.xml](https://github.com/Automattic/uCrop/blob/develop-non-native/sample/src/main/AndroidManifest.xml) file within the `sample` module.